packages/fxa-content-server/app/tests/spec/lib/metrics.js

```diff
@@ -76,12 +76,13 @@ describe('lib/metrics', function() {
   });
 
   it('has the expected notifications', () => {
-    assert.lengthOf(Object.keys(metrics.notifications), 7);
+    assert.lengthOf(Object.keys(metrics.notifications), 8);
 
     assert.isTrue('flow.initialize' in metrics.notifications);
     assert.isTrue('flow.event' in metrics.notifications);
     assert.isTrue('set-email-domain' in metrics.notifications);
     assert.isTrue('set-sync-engines' in metrics.notifications);
+    assert.isTrue('set-plan-and-product-id' in metrics.notifications);
```

Comment:
Looks like we also need a test to assert that this actually sets the ids correctly?

Reply:
Yup, added.